app/helpers/home_helper.rb

```diff
@@ -0,0 +1,2 @@
+module HomeHelper
```

Comment:
@agallant121  what are these HomeHelper and MeHelper modules doing? It doesn't look like there is anything in here